### PR TITLE
Increase time-out in OngoingStreamAuthIT

### PR DESCRIPTION
**Pull request by pbatko-da**

### CODE REVIEW

File: ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala [hunk 1]
```diff
@@ -101,7 +101,7 @@ final class OngoingStreamAuthIT
           new AssertionError("Timed-out waiting while waiting for stream to abort")
         )
       }
-      _ = new Timer(true).schedule(timerTask, 100)
+      _ = new Timer(true).schedule(timerTask, 1000)
```
Comment (kamil-da) [inline ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala:104]: Is using the same timeout for observing stream termination and the cache expiry intentional? Could you clarify what happens if the termination doesn't fits in this 1 second period? Would it race with the cache expiry timeout in some way?

Comment (pbatko-da) [reply ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala:104]: There's a thread sleep just before this timer and after granting a new right:
```
_ = Thread.sleep(UserManagementCacheExpiryInSeconds.toLong * 1000)
```
This timeout is on-top of the above one. Does it clarify you concerns?

Comment (pbatko-da) [reply ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala]: Changed the code, please take another look

Comment (kamil-da) [reply ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala]: 🤦  You're obviously right - it was 2 seconds delay from the start.
It looks clearer now though. Thank you!

**Review by kamil-da — APPROVED** [review]

File: ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala [hunk 2]
```diff
@@ -86,25 +86,22 @@ final class OngoingStreamAuthIT
...
         userId = userIdAlice,
         Right(Right.Kind.CanActAs(Right.CanActAs(UUID.randomUUID().toString))),
       )
-      _ = Thread.sleep(UserManagementCacheExpiryInSeconds.toLong * 1000)
-      //
-      // Timer that finishes the stream in case it isn't aborted as expected
-      timerTask = new TimerTask {
-        override def run(): Unit = streamObserver.onError(
-          new AssertionError("Timed-out waiting while waiting for stream to abort")
+      _ <- Delayed.Future.by(2.second)(
```
Comment (mziolekda) [inline ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala:96]: Why not make it relative to `UserManagementCacheExpiryInSeconds` size i.e. either
`UserManagementCacheExpiryInSeconds*2` or `UserManagementCacheExpiryInSeconds+1`?

Comment (pbatko-da) [reply ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala]: Changed to `UserManagementCacheExpiryInSeconds + 1`

**Pull request merged**